aurBuild.sh

```diff
@@ -1,7 +1,21 @@
 #!/bin/bash
-# Author: Sjon Hortensius <sjon@hortensius.net>
-# build a package from aur. Extra options will be passed to makepkg
-# usage: aurBuild [--install] [package]
+if [[ "$1" == "--help" ]]; 
```

Comment:
';' is an alternative for a newline; eg if [[ $a ]]; then echo $a; done. But you have a newline here so it's redundant